airflow/www/templates/airflow/dag.html

```diff
@@ -139,6 +139,9 @@ <h4 class="pull-right" style="user-select: none;-moz-user-select: auto;">
           <li><a href="{{ url_for('Airflow.code', dag_id=dag.dag_id, root=root) }}">
             <span class="material-icons" aria-hidden="true">code</span>
             Code</a></li>
+          <li><a href="{{ url_for('Airflow.audit_log', dag_id=dag.dag_id) }}">
+            <span class="material-icons" aria-hidden="true">security</span>
```

Comment:
I'm not sure if `security` is the best icon to use here.

I kind of like `plagiarism` https://fonts.google.com/icons?selected=Material%20Icons%3Aplagiarism%3A